src/tsc/tsc.ts

```diff
@@ -132,6 +132,11 @@ namespace ts {
         const commandLineOptions = commandLine.options;
         if (configFileName) {
             const configParseResult = parseConfigFileWithSystem(configFileName, commandLineOptions, sys, reportDiagnostic)!; // TODO: GH#18217
+            if (commandLineOptions.showConfig) {

```

Comment:
Interesting. Should we intentionally be ignoring this in --b mode? Should it error when invoked without tsconfig file?

Reply:
I think it's fine without a config file - serves as a fine way to convert a response file or other command line args to a tsconfig.

Reply:
Afaik it is ignored in `-b` mode; iirc that bails out way early on in the command line parser, since it has its own subset of args.

Reply:
I think we could have this even in --b just like how we have eg (list files, list emitted files) could be good information I think. Especially in verbose mode?